.github/workflows/nuget-audit.yml

```diff
@@ -0,0 +1,24 @@
+name: NuGet Audit
+on:
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+defaults:
+  run:
+    shell: bash
```

Comment:
`bash` is the default shell for non-Linux runners already, so we don't need to set this.